git/commands.md

```diff
@@ -34,6 +35,7 @@ git add -p       # show new/changed files one by one with their changes and a qu
 
 ##### Removing changes from the index
 ```
+
 git reset            # remove from the index all changes added to it (in the working directory all changes will be saved), the antipode of git add
```

Comment:
The description of the command is better to provide before the block of commands